yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@adobe/css-tools@^4.0.1":
   version "4.1.0"
-  resolved "https://registry.yarnpkg.com/@adobe/css-tools/-/css-tools-4.1.0.tgz#417fef4a143f4396ad0b3b4351fee21323f15aa8"
+  resolved "https://registry.npmjs.org/@adobe/css-tools/-/css-tools-4.1.0.tgz"
```

Comment:
I don't think there should be any changes made to the `yarn.lock`-file in this PR.